### ISSUE TITLE
Open code of conduct is not "open"

### ISSUE THREAD

**Issue by sstewartgallus** [todogroup/opencodeofconduct#103]
This code of conduct has been indefinitely paused and is not listening to the opinions of the community. This is not an open code of conduct.

**Comment by metabench** [comment]
I agree. It's been a while since @bkeepers said 'We are consulting with stakeholders, community leaders, and legal professionals. We want to make sure any further changes are done carefully and thoughtfully'. There has not been any discussion on this Github forum, that discussion seems to have taken place privately rather than implied by the name 'Talk Openly Develop Openly'.

The Internet at large has not heard about the conclusions reached by that core group - perhaps it would be a better idea to open up the discussion in line with the stated purpose of the organization.
